doc/api/fs.md

```diff
@@ -4697,6 +4553,94 @@ The following constants are meant for use with the [`fs.Stats`][] object's
   </tr>
 </table>
 
+## File System Flags
+
+The following flags are available wherever the `flag` option takes a
+string:
+
+* `'a'` - Open file for appending.
+  The file is created if it does not exist.
+
+* `'ax'` - Like `'a'` but fails if the path exists.
+
+* `'a+'` - Open file for reading and appending.
+  The file is created if it does not exist.
+
+* `'ax+'` - Like `'a+'` but fails if the path exists.
+
+* `'as'` - Open file for appending in synchronous mode.
+  The file is created if it does not exist.
+
+* `'as+'` - Open file for reading and appending in synchronous mode.
+  The file is created if it does not exist.
+
+* `'r'` - Open file for reading.
+  An exception occurs if the file does not exist.
+
+* `'r+'` - Open file for reading and writing.
+  An exception occurs if the file does not exist.
+
+* `'rs+'` - Open file for reading and writing in synchronous mode. Instructs
+  the operating system to bypass the local file system cache.
+
+  This is primarily useful for opening files on NFS mounts as it allows	
```

Comment:
Md-lint finds Trailing tab here)

Reply:
Resolved in 4b708196caa8ee6115238326b658e8ff8f280caf